.github/workflows/monorepo-split-alpha.yaml

```diff
@@ -0,0 +1,28 @@
+on:
+    push:
+        branches:
+            - 'alpha'
+name: Split alpha branch
+permissions:
+    contents: write
+    actions: write
+jobs:
+    split-branch:
+        runs-on: ubuntu-22.04
+        name: Split ${{ github.ref_name }} branch
+        steps:
+            -   name: GIT checkout branch - ${{ github.ref_name }}
+                uses: actions/checkout@v4
+                with:
+                    ref: ${{ github.ref }}
+            -   name: Ensure ${{ github.ref_name }} branch dependencies in project-base/composer.json
+                run: |
+                    sed -i 's/"14.0.x-dev"/"dev-${{ github.ref_name }} as 14.0.x-dev"/' project-base/app/composer.json
```

Comment:
Can we use ${{ github.event.repository.default_branch }} instead of hardcoding 14.0?

Reply:
referenced in a new issue – will be improved separately